flow/workflows/cdc_flow.go

```diff
@@ -506,8 +506,7 @@ func CDCFlowWorkflowWithConfig(
 			RetryPolicy: &temporal.RetryPolicy{
 				MaximumAttempts: 20,
 			},
-			SearchAttributes:    mirrorNameSearch,
-			WaitForCancellation: true,
```

Comment:
Turning off WaitForCancellation will hide children workflows taking too long to cancel themselves

At least for source connector we'll be waiting for the disconnect anyways to drop slot

Reply:
What do you mean by hide?

Reply:
I mean the workflow will say it's canceled, which seems good, but we shouldn't really consider it canceled until these waits are satisfied